### PR TITLE
3scale backup

### PR DESCRIPTION
**Pull request by aidenkeating**
Sets up CronJobs to backup 3scale's MySQL, Postgres and Redis instances.

Verification:
- Create a secret in `default` named `s3-credentials` that contains your credentials
- Update the master URL in `managed.template` and set `core_install=false` if you've already setup integreatly
- Set `backup_schedule` in `manifest.yaml` to `'*/1 * * * *'` to run each minute
- Run: `ansible-playbook -i inventories/managed.template playbooks/managed/install.yml`
- Wait a minute
- In the `3scale` namespace
  * Ensure the Postgres backup Pod succeeds
  * Ensure the MySQL backup Pod succeeds
  * Ensure the Redis backup Pod succeeds
- Check S3 and ensure the archives have been pushed up.
- Delete the CronJobs

### CODE REVIEW

**Comment by aidenkeating** [comment]
@pb82 @maleck13 Mind taking a look?

**Comment by pb82** [comment]
:eyes:

File: evals/inventories/group_vars/all/manifest.yaml [hunk 1]
```diff
@@ -91,3 +92,10 @@ middleware_monitoring_operator_resources: 'https://raw.githubusercontent.com/int
 
 msbroker_release_tag: 'v0.0.1'
 msbroker_template: 'https://raw.githubusercontent.com/integr8ly/managed-service-broker/{{ msbroker_release_tag }}/templates/broker.template.yaml'
+
+# information about backups
+backup_version: master
+backup_image_tag: latest
```
Comment (odra) [inline evals/inventories/group_vars/all/manifest.yaml]: should be master as well

**Comment by aidenkeating** [comment]
@pb82 Looks like I accidentally dismissed your review when changing the image tag from `latest` to `master`, they both point to the same image. Mind taking another quick look?

File: evals/inventories/group_vars/all/manifest.yaml [hunk 2]
```diff
@@ -91,3 +92,10 @@ middleware_monitoring_operator_resources: 'https://raw.githubusercontent.com/int
 
 msbroker_release_tag: 'v0.0.1'
 msbroker_template: 'https://raw.githubusercontent.com/integr8ly/managed-service-broker/{{ msbroker_release_tag }}/templates/broker.template.yaml'
+
+# information about backups
+backup_version: master
+backup_image_tag: master
```
Comment (mikenairn) [inline evals/inventories/group_vars/all/manifest.yaml]: Do we intend to create a release of this? This needs to be done before we create 1.3 anyway

Comment (odra) [reply evals/inventories/group_vars/all/manifest.yaml]: the repository is using the same travis file that other projects such as our operators use

Comment (mikenairn) [reply evals/inventories/group_vars/all/manifest.yaml]: OK. What i'm saying is this shouldn't be using master here, but an actual release tag like everything else.

Comment (odra) [reply evals/inventories/group_vars/all/manifest.yaml]: got it but the repository was created last week so there isn't a release of it yet - should we create one for the next release (1.3.0)?

Comment (mikenairn) [reply evals/inventories/group_vars/all/manifest.yaml]: It was really just a comment to make sure we do create a release and update this to an explicit tag before we create 1.3 (Not trying to stop this PR from being merged). Maybe there is still work going on in here that makes this not desirable at the moment and we want to use master for a while, that's up to you. Just needs to be fixed before 1.3 is created IMO.

Comment (aidenkeating) [reply evals/inventories/group_vars/all/manifest.yaml]: @mikenairn @odra I'd be in favour of holding off on a release until we've confirmed there's no issues with backups on other components. I can create an issue in the Epic to ensure we do that if that sounds good to you guys?

Comment (odra) [reply evals/inventories/group_vars/all/manifest.yaml]: agreed

Comment (aidenkeating) [reply evals/inventories/group_vars/all/manifest.yaml]: INTLY-1222 has been created to address this before the 1.3 release

**Pull request merged**